packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -189,7 +189,7 @@ export class AtomicSearchBox implements InitializableComponent<Bindings> {
    * When customizing the dimensions of the textarea element using the `"textarea"` CSS part, it is important to also apply the styling to its container's ::after pseudo-element as well as the `"textarea-spacer"` part.
    * The buttons within the search box are likely to need adjusting as well.
    *
-   * Example:
+   * @example
```

Comment:
@jpmarceau 👁️

Reply:
Sadly, we don't do anything with `example` tags at the moment. So, merging this would cause the examples to disappear from the docs, if we did it right now 🙃 

We could adjust our generation scripts to handle that; I don't think it'll take long, but we'd need to plan for it. Maybe it would make more sense to:
1- revert the changes here
2- my team reworks out scripts to handle the `example` tags
3- my team revisits the atomic package to use `example` tags, like you're doing in this PR?

Reply:
Arf, yes ok.
There polluting the Storybook description but we'll live with that for the time being 😅

Reply:
Reverted in [82fc00b](https://github.com/coveo/ui-kit/pull/3980/commits/82fc00b09c9cfc39adbdb12ae92e756263bb7819)

Reply:
Ok if they're polluting storybook, then I'll make sure we prioritize
created https://coveord.atlassian.net/browse/DOC-15067